File: js/q20.js
var arr=[80,20,30,40,50]
var max=arr[0]
var min=arr[0]

for(i=0;i<arr.length;i++){
    if(min>arr[i]){
        min=arr[i]
    }
    if(max<arr[i]){
        max=arr[i]
    }
}
var diff=max-min

console.log("The difference between maximum and minimum is",diff)

File: clangua/q20.c
#include <stdio.h>
#include <string.h>

int main() {
    
    int arr[]={80,30,70,50,20};
    int i;
    int max=arr[0];
    int min=arr[0];
    int len=sizeof(arr)/sizeof(arr[0]);
    for(i=0;i<len;i++){
        if(min>arr[i]){
            min=arr[i];
        }
        if(max<arr[i]){
            max=arr[i];
        }
    }
    int diff=max-min;
    
    printf("The difference between minimum and maximum is %d",diff);

    return 0;
}